app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -1,14 +1,31 @@
 import React from "react";
-import { Checkbox } from "neetoui";
+import { Checkbox, Avatar, Badge, Button } from "neetoui";
+
+const GetBadgeColor = tag => {
+  switch (tag) {
+  case "Agile Workflow":
+    return "green";
+  case "Bug":
+    return "red";
+  default:
+    return "blue";
+  }
+};
 
 export default function NoteTable({
   selectedNoteIds,
   setSelectedNoteIds,
+  setShowDeleteAlert,
   notes = [],
 }) {
+  const handleDeleteNote = noteId => {
+    setSelectedNoteIds([noteId]);
+    setShowDeleteAlert(true);
+  };
+
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
```

Comment:
Please add classname `nui-table--hover` to add hover effects to the table row.

Reply:
@karthiknmenon I checked `nui-table--hover` has no further effect after `nui-table--actions`. Do you still want me to add class anyway?

Reply:
@vipul-goel Is there hover effects (background color to grey) on hovering over a row on the table?

Reply:
@karthiknmenon yes there are gray background hover effects by only using `nui-table--actions`